pmaudit

```diff
@@ -506,12 +515,17 @@ def main():
         else:
             opts = parser.parse_args()
             cfglog(opts.verbosity)
-            cmd = [os.getenv('SHELL', '/bin/sh')]
```

Comment:
I think you've lost the initialization here:

     cmd = [opts.shell]

Not sure how this worked as is.

Reply:
You're right.  Re-inserting.  In the longer term this needs a test suite.